packages/gateway/tests/integration/connection.spec.ts

```diff
@@ -120,7 +119,7 @@ const createUws = async (options: {
         }
         // retrieve listening port
         uwsToken = token
-        port = uWS.us_socket_local_port(token)
+        port = uWS.us_socket_local_port(uwsToken)
```

Comment:
```suggestion
        port = uWS.us_socket_local_port(token)
```

Reply:
For some reason GitHub doesn't allow me to select three lines and make one suggestion, so please check the one above and the one below

Reply:
This causes an issue with ESLint. Since token is any (really good typings) eslint will error with: "Unsafe argument of type `any` assigned to a parameter of type `us_socket`." But with an `as` it can be used